Assets/Mirror/Runtime/NetworkManager.cs

```diff
@@ -623,7 +627,7 @@ public virtual void OnServerReady(NetworkConnection conn)
             NetworkServer.SetClientReady(conn);
         }
 
-        public virtual void OnServerAddPlayer(NetworkConnection conn, NetworkReader extraMessageReader)
+        public virtual void OnServerAddPlayer(NetworkConnection conn, NetworkMessage extraMessage)
```

Comment:
Do we need the NetworkConnection anymore?  it comes in the message

Reply:
other than that,  this is good

Reply:
in the long run it might make sense to keep it. passing NetworkMessage is better than NetworkReader, but ideally we'd just pass the MessageBase some day. right now the passed NetworkMessage doesn't have conn/msgtype anyway - only the reader to create the MessageBase.